### PR TITLE
Create base env script

### PR DESCRIPTION
**Pull request by thepetk**
This PR is just a suggestion to create a `base` env to host all the default env var values used across different samples.

Thought it might be a good addition, as it makes it easier for someone to add a new env var in one of the samples (new or existing one). More specifically, when someone needs to differentiate a sample env var value from all the others:
* They should declare the env var in the env file of the sample they want to differentiate.
* They just need to export the default value in the `scripts/envs/base` instead of exporting in all the files inside `scripts/envs/` dir.

An example that the suggested feature could help can be found in the RAG related [PR](https://github.com/redhat-ai-dev/ai-lab-template/pull/27/files) where it's necessary to declare the `SUPPORT_DB` env var as `false` in all the sample env files. Following the suggested workaround one has to declare it only in the `scripts/envs/base` (default value) and in the `rag` env var (value used for this sample).

Finally, an update is made to the `scripts/import-ai-lab-samples` file to first source the `base` env var values and then source the sample's env vars. This way if someone wants to override a default value they just need to declare it in the sample's env file.

### CODE REVIEW

File: scripts/envs/base
Comment (yangcao77) [inline scripts/envs/base:9]: can be set to all false. and only set to true in each particular env file

File: scripts/envs/chatbot
Comment (yangcao77) [inline scripts/envs/chatbot:6]: for all the env files, can we have a comment at the top? indicating the base env value is under `./env` and can be override in this file?

**Review by yangcao77 — COMMENTED** [review]
Generally looks good. some minor comments

File: scripts/envs/chatbot
Comment (thepetk) [reply scripts/envs/chatbot:9]: @yangcao77 I think is a good idea. I've added a comment in every env file.

**Review by yangcao77 — DISMISSED** [review]

**Comment by thepetk** [comment]
As this PR is of low importance compared to https://github.com/redhat-ai-dev/ai-lab-template/pull/27 I'll wait until the RAG PR is merged and then I'll rebase & merge.

**Comment by thepetk** [comment]
@yangcao77 @maysunfaisal I've updated the PR after the merge of RAG sample updates

**Review by maysunfaisal — APPROVED** [review]

**Pull request merged**
